tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
+# | 5        | smiley face (catch trial)                                |
+# +----------+----------------------------------------------------------+
+# | 32       | subject button press                                     |
+# +----------+----------------------------------------------------------+
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
+
+###############################################################################
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data; the ``/`` character in the dictionary keys allows pooling
+# across conditions by requesting partial condition descriptors (i.e.,
+# requesting ``'auditory'`` will select all epochs with Event IDs 1 and 2;
+# requesting ``'left'`` will select all epochs with Event IDs 1 and 3). An
+# example of this is shown in the next section. There is also a convenient
+# :func:`~mne.viz.plot_events` function for visualizing the distribution of
+# events across the duration of the recording (to make sure event detection
+# worked as expected). Here we'll also make use of the :class:`~mne.Info`
+# attribute to get the sampling frequency of the recording (so our x-axis will
+# be in seconds instead of in samples).
+
+fig = mne.viz.plot_events(events, event_id=event_dict, sfreq=raw.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+###############################################################################
+# Epoching continuous data
+# ^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Define parameters:
-
-import numpy as np  # noqa
-n_cycles = 2  # number of cycles in Morlet wavelet
-freqs = np.arange(7, 30, 3)  # frequencies of interest
-
-##############################################################################
-# Compute induced power and phase-locking values and plot gradiometers:
-
-from mne.time_frequency import tfr_morlet  # noqa
-power, itc = tfr_morlet(epochs, freqs=freqs, n_cycles=n_cycles,
-                        return_itc=True, decim=3, n_jobs=1)
-power.plot([power.ch_names.index('MEG 1332')])
-
-##############################################################################
-# Inverse modeling: MNE and dSPM on evoked and raw data
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The :class:`~mne.io.Raw` object and the events array are the bare minimum
+# needed to create an :class:`~mne.Epochs` object, which we create with the
+# :class:`mne.Epochs` class constructor. Here we'll also specify some data
+# quality constraints: we'll reject any epoch where peak-to-peak signal
+# amplitude is beyond reasonable limits for that channel type. This is done
+# with a *rejection dictionary*:
+
+reject_criteria = dict(mag=4000e-15,     # 4000 fT
+                       grad=4000e-13,    # 4000 fT/cm
+                       eeg=150e-6,       # 150 μV
+                       eog=250e-6)       # 250 μV
```

Comment:
I would add a note that these numbers of dataset specific and cannot used blindly. Eventually pointing out to the autoreject paper that aims to address this issue.